### ISSUE TITLE
2.1.1 fails to build due to missing symbol (libpdalpython.so: undefined symbol: _ZNK4pdal16PipelineExecutor11getMetadataEv)

### ISSUE THREAD

**Issue by sebastic** [PDAL/python#19]
With the fix from #18 the Python bindings still fail to build:
```
running build_ext
copying /build/python-pdal-2.1.1+ds/.pybuild/cpython2_2.7_pdal/build/pdal/libpdalpython.so -> pdal
Traceback (most recent call last):
  File "setup.py", line 218, in <module>
    setup(ext_modules=extensions, **setup_args)
  File "/usr/lib/python2.7/dist-packages/setuptools/__init__.py", line 140, in setup
    return distutils.core.setup(**attrs)
  File "/usr/lib/python2.7/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/usr/lib/python2.7/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "/usr/lib/python2.7/dist-packages/setuptools/command/test.py", line 228, in run
    self.run_tests()
  File "/usr/lib/python2.7/dist-packages/setuptools/command/test.py", line 250, in run_tests
    exit=False,
  File "/usr/lib/python2.7/unittest/main.py", line 94, in __init__
    self.parseArgs(argv)
  File "/usr/lib/python2.7/unittest/main.py", line 149, in parseArgs
    self.createTests()
  File "/usr/lib/python2.7/unittest/main.py", line 158, in createTests
    self.module)
  File "/usr/lib/python2.7/unittest/loader.py", line 130, in loadTestsFromNames
    suites = [self.loadTestsFromName(name, module) for name in names]
  File "/usr/lib/python2.7/unittest/loader.py", line 91, in loadTestsFromName
    module = __import__('.'.join(parts_copy))
  File "/build/python-pdal-2.1.1+ds/test/__init__.py", line 3, in <module>
    from test.test_pipeline import test_suite
  File "/build/python-pdal-2.1.1+ds/test/test_pipeline.py", line 2, in <module>
    import pdal
  File "/build/python-pdal-2.1.1+ds/pdal/__init__.py", line 3, in <module>
    from .pipeline import Pipeline
  File "/build/python-pdal-2.1.1+ds/pdal/pipeline.py", line 2, in <module>
    from pdal import libpdalpython
ImportError: /build/python-pdal-2.1.1+ds/pdal/libpdalpython.so: undefined symbol: _ZNK4pdal16PipelineExecutor11getMetadataEv
```

This may be caused by the PDAL build not using `-D_GLIBCXX_USE_CXX11_ABI=0` as the symbols file for libpdal-base has:
```
_ZNK4pdal16PipelineExecutor11getMetadataB5cxx11Ev@Base
```

**Comment by hobu** [comment]
so we should NOT be building with `-D_GLIBCXX_USE_CXX11_ABI=0`?

**Comment by hobu** [comment]
Other users are reporting ABI mixing issues, and I don't know how to discriminate.

**Comment by sebastic** [comment]
I think that you need to build both pdal & python-pdal with `-D_GLIBCXX_USE_CXX11_ABI=0, or neither.

Now pdal has been built with the C++11 ABI, but python-pdal has not.

I'll do a test build of pdal with `-D_GLIBCXX_USE_CXX11_ABI=0` to see if it fixes this issue.

**Comment by hobu** [comment]
It might only be the conda gcc 4.8.2 builds that have this problem.

**Comment by hobu** [comment]
2.1.3 pushed that tries to do D_GLIBCXX_USE_CXX11_ABI for only 4.8 conda.

**Comment by sebastic** [comment]
Confirmed fixed in 2.1.3.
